### PR TITLE
remove unused legacy eventbrite code + add 2021 online burn ticket url

### PR DESCRIPTION
**Pull request by 0xdevalias**
fixes https://github.com/sparkletown/sparkle/issues/1390 (raised by @yarikoptic) as per my exploration in https://github.com/sparkletown/sparkle/issues/1390#issuecomment-841591461

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
     "react-dnd": "^11.1.3",
     "react-dnd-html5-backend": "^11.1.3",
     "react-dom": "^16.13.1",
-    "react-eventbrite-popup-checkout": "^1.0.5",
```
Comment (0xdevalias) [inline package.json:54]: [note] This didn't appear to be used anymore.

File: public/index.html [hunk 2]
```diff
@@ -71,7 +71,6 @@
     />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Sparkle</title>
-    <script src="https://www.eventbrite.co.uk/static/widgets/eb_widgets.js"></script>
```
Comment (0xdevalias) [inline public/index.html:74]: [note] This didn't appear to be used anymore.

File: src/react-app-env.d.ts [hunk 3]
```diff
@@ -1,12 +1 @@
...
-declare module "react-eventbrite-popup-checkout" {
-  interface PropType {
-    ebEventId: string;
-    className: string;
-    isModal?: boolean;
-    onOrderComplete?: () => void;
-  }
-  const EventbriteButton: React.FunctionComponent<PropType>;
-  export default EventbriteButton;
-}
```
Comment (0xdevalias) [inline src/react-app-env.d.ts:12]: [note] This relates to the removed dependency, which didn't appear to be used anymore.

File: src/settings.ts [hunk 4]
```diff
@@ -59,6 +59,8 @@ export const VENUE_NAME_MIN_CHAR_COUNT = 3;
 export const VENUE_NAME_MAX_CHAR_COUNT = 50;
 export const PLAYA_VENUE_NAME = "Jam";
 export const PLAYA_VENUE_ID = "jamonline";
+export const GIFT_TICKET_MODAL_URL =
+  "https://here.burningman.org/event/virtualburn";
```
Comment (0xdevalias) [inline src/settings.ts:63]: [note] This is the URL for the 2021 online burn, taken from the link on http://sparklever.se/

**Review by 0xdevalias — COMMENTED** [review]
[notes] Some notes on this PR to make review easier.

File: src/settings.ts (hunk 4 above)
Comment (yarikoptic) [reply src/settings.ts:63]: I wonder, why to keep any specific event vendoring in the main/staging version and not just replace with "" so too be customized for a given event?

Comment (0xdevalias) [reply src/settings.ts:63]: Minimising the scope of changes for this PR. The better long term fix would be to load this from configure the database, but that’s OOS here.

Comment (yarikoptic) [reply src/settings.ts:63]: +1 on centralizing event specific configuration (later) and minimizing the score of the pr. 
What will need to be done for ohbm (in env/ohbm ?) to not trigger any eventbrite related functionality/calls?

Comment (0xdevalias) [reply src/settings.ts:63]: Nothing, this PR removes everything eventbrite/related except for this static string.

**Review by sunny-viktoryia — APPROVED** [review]
Looks good. :upside_down_face:

**Pull request merged**